core/modules/proxy/mysqlProxy.lua

```diff
@@ -281,8 +281,7 @@ function queryType()
            string.find(qU, "^LOAD ") or
            string.find(qU, "^CREATE ") or
            string.find(qU, "^ALTER ") or
-           string.find(qU, "^TRUNCATE ") or
-           string.find(qU, "^DROP ") then
+           string.find(qU, "^TRUNCATE ") then
             err.set(ERR_NOT_SUPPORTED,
                     "Sorry, this type of queries is disallowed.")
```

Comment:
This is unrelated to the ticket work but it would probably be nice to update this to something nicer like "Sorry, this type of query is not supported."